test/fuzzer/duckfuzz/default_duckfuzz_values.test

```diff
@@ -0,0 +1,14 @@
+# name: test/fuzzer/duckfuzz/default_duckfuzz_values.test
```

Comment:
Could we remove this test? Running any fuzzing during regular tests always has a chance of failing or blowing up randomly.